sdk/src/test/java/com/silanis/esl/sdk/builder/DocumentPackageSettingsBuilderTest.java

```diff
@@ -14,113 +14,125 @@
         assertFalse("disableOptOutOther was not set correctly", documentPackageSettings.getDisableOptOutOther());
 
-        documentPackageSettings = newDocumentPackageSettings().withOptOutReason("reason").build();
-        assertThat("Builder returned a null object", documentPackageSettings, notNullValue());
+        documentPackageSettings = newDocumentPackageSettings().withOptOutReason(STR_REASON).build();
+        assertThat(NULL_ERROR_MSG, documentPackageSettings, notNullValue());
         assertThat("optOutReasons was not set correctly", documentPackageSettings.getOptOutReasons(), hasSize(1));
-        assertThat("optOutReasons was not set correctly", documentPackageSettings.getOptOutReasons().get(0), is("reason"));
+        assertThat("optOutReasons was not set correctly", documentPackageSettings.getOptOutReasons().get(0), is(STR_REASON));
 
         documentPackageSettings = newDocumentPackageSettings().withoutWatermark().build();
-        assertThat("Builder returned a null object", documentPackageSettings, notNullValue());
+        assertThat(NULL_ERROR_MSG, documentPackageSettings, notNullValue());
         assertTrue("hideWatermark was not set correctly", documentPackageSettings.getHideWatermark());
 
         documentPackageSettings = newDocumentPackageSettings().withWatermark().build();
-        assertThat("Builder returned a null object", documentPackageSettings, notNullValue());
+        assertThat(NULL_ERROR_MSG, documentPackageSettings, notNullValue());
         assertFalse("hideWatermark was not set correctly", documentPackageSettings.getHideWatermark());
 
         documentPackageSettings = newDocumentPackageSettings().withEnforceCaptureSignature().build();
-        assertThat("Builder returned a null object", documentPackageSettings, notNullValue());
+        assertThat(NULL_ERROR_MSG, documentPackageSettings, notNullValue());
         assertTrue("enforceCaptureSignature was not set correctly", documentPackageSettings.getEnforceCaptureSignature());
 
         documentPackageSettings = newDocumentPackageSettings().withAda().build();
-        assertThat("Builder returned a null object", documentPackageSettings, notNullValue());
+        assertThat(NULL_ERROR_MSG, documentPackageSettings, notNullValue());
         assertTrue("ada was not set correctly", documentPackageSettings.getAda());
+
+        documentPackageSettings = newDocumentPackageSettings().enableTimeBasedExpiry().build();
+        assertThat(NULL_ERROR_MSG, documentPackageSettings, notNullValue());
+        assertTrue("Time base expiry was could not be enabled..", documentPackageSettings.getEnableTimeBasedExpiry());
+
+        documentPackageSettings = newDocumentPackageSettings().enableTimeBasedExpiry().withRemainingDays(10).build();
+        assertThat(NULL_ERROR_MSG, documentPackageSettings, notNullValue());
+        assertThat("Expiry days was not set correctly", documentPackageSettings.getRemainingDays().equals(Integer.valueOf(10)));
```

Comment:
Please use a Matcher like other assert statements.

ex)
`assertThat("Expiry days was not set correctly", documentPackageSettings.getRemainingDays(), is(10));`

Reply:
done